src/util/hass-call-api.ts

```diff
@@ -53,13 +57,16 @@ export default async function hassCallApi<T>(
   method: string,
   path: string,
   parameters?: Record<string, unknown>,
-  headers?: Record<string, string>
+  headers?: Record<string, string>,
+  signal?: AbortSignal,
+  returnResponse?: boolean
```

Comment:
Shouldn't we create a dedicated function for that instead of extending this one?

Reply:
:+1: oh yes, it was actually on my list but I forgot it